fhir-validation/src/test/java/com/ibm/fhir/validation/test/ObservationProfileTest.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * (C) Copyright IBM Corp. 2020
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.validation.test;
+
+
+import static com.ibm.fhir.path.util.FHIRPathUtil.compile;
+import static org.testng.Assert.assertEquals;
+
+import java.util.List;
+
+import org.testng.annotations.Test;
+
+import com.ibm.fhir.model.annotation.Constraint;
+import com.ibm.fhir.model.resource.StructureDefinition;
+import com.ibm.fhir.profile.ConstraintGenerator;
+import com.ibm.fhir.registry.FHIRRegistry;
+
+public class ObservationProfileTest {
+    @Test
+    public void testObservationProfile() throws Exception {
+        StructureDefinition profile = FHIRRegistry.getInstance().getResource("http://ibm.com/fhir/StructureDefinition/my-observation",  StructureDefinition.class);
+        ConstraintGenerator generator = new ConstraintGenerator(profile);
+        List<Constraint> constraints = generator.generate();
+        constraints.forEach(System.out::println);
```

Comment:
does it generate a lot of output?

Reply:
It outputs a single line to the console.